members/ranks/daoists.md

```diff
@@ -1,14 +1,16 @@
 # DAOists
 
-DAOists are the caretakers of the server! They have multiple roles, each with their own unique blend of values, principles, skills and ideas. Different people are here for different reasons. Some are here for neutrality or to balance discussions, some to provide guidance whether it be technical or spiritual motivation, some to spread their knowledge and educate others, some to build things they believe in and support their vision for the future, some to connect everyone in DOTSAMA and so on.
+DAOists are the caretakers of the server. They have multiple roles, each with their own unique blend of values, principles, skills and ideas. Different people are here for different reasons. Some are here for neutrality or to balance discussions, some to provide guidance whether it be technical or spiritual motivation, some to spread their knowledge and educate others, some to build things they believe in and support their vision for the future, some to connect everyone in DOTSAMA and so on.
```

Comment:
Should we remove DOTSAMA here now that it's being widely frowned upon by Parity? Perhaps replace it with Polkadot and Kusama or "the Polkadot ecosystem"

Reply:
well that depends on philosophy perhaps ;)

When Parity began pushing that narrative, I got the feeling some people wanted to be rebels and continue using it. Not sure if our community completely fall in line since we are much more involved in the Kusama community chain, so we wouldn't want to be cast off and left to drift ...

and besides, @bacchist wrote that part ;)

Reply:
✅